### ISSUE TITLE
gnome-pomodoro-0.20.0 notifications don't work under Gnome/Wayland

### ISSUE THREAD

**Issue by logrusx** [gnome-pomodoro/gnome-pomodoro#616]
## Issue Summary

When pomodoro starts or nears its end, notifications are shown, but when on Wayland, they do not appear.

## Steps to Reproduce

1. Start a Gnome Wayland session
2.
a) Start pomodoro
b) Wait for the last minute of pomodoro.

## Context (logs, error output, screencast, OS, Desktop  etc)

Gnome 41/Wayland

## Expected Behavior

a) A notification should appear
b) A notification with the option to extend the pomodoro should appear

## Current Behavior

No notifications are shown

## Other information

Notifications sometimes appear but I've been unable to identify any logical sequence that causes or prevents it. Most of the time they don't in fact both notifications appeared only once, while I was testing #584 as it seem related to another issue with another extension.

For now it seems under Wayland, the first notification appears only, and it appears only immediately after login or screen poweroff/lock timeout adjustment and no other applications running.

**Comment by kamilprusko** [comment]
Showing this notification interferes with Do Not Disturb feature in GNOME Shell. I'm surprised it works more often than not :D

Pomodoro extension would need to implement its own version of this feature. Hackish, but needs to be done.

**Comment by logrusx** [comment]
`Showing this notification interferes with Do Not Disturb feature in GNOME Shell.`
Thanks for letting me know. It this some option to configure or a Gnome feature that's in general conflict with notifications? Because I haven't configured/enabled any such option explicitly.
`I'm surprised it works more often than not :D`
It doesn't. Only in the described cases :-)
